server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -65,6 +65,112 @@ impl From<messageattempt::Model> for MessageAttemptOut {
     }
 }
 
+// FIXME: Contains all members of a [`v1::endpoints::message::MessageOut`], so find a way to
+// #[serde(flatten)]` a [`super::message::MessageOut`] in cleanly. An attempt was made, but it would
+// require a custom [`ModelOut`] impl and another impl of [`sea_orm::TryGettable`] on the
+// [`super::message::MessageOut`] type in order for [`FromQueryResult`] to work.
+
+/// A model containing information on a given message plus additional fields on the last attempt for
+/// that message.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, ModelOut, FromQueryResult)]
```

Comment:
```suggestion
#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, ModelOut)]
```
No longer needed, no?